astroquery/esa/jwst/core.py

```diff
@@ -83,6 +83,15 @@ def __init__(self, *, tap_plus_handler=None, data_handler=None, show_messages=Tr
         if show_messages:
             self.get_status_messages()
 
+    def __check_list_strings(self, list):
```

Comment:
I would strongly advise calling the parameter something else than `list` to avoid overriding the `list()`